CHANGELOG.md

```diff
@@ -0,0 +1,26 @@
+# Change Log
+
+`healthvault-ios-sdk` adheres to [Semantic Versioning](http://semver.org/).
+
+#### 3.x Pre-release
+- `3.0-preview` - [3.0-preview](#30-preview) | [3.0-preview.2](#30-preview2) | [3.0-preview.3](#30-preview3) | [3.0-preview.4](#30-preview4)
+
+---
+## [3.0-preview.4](https://github.com/Microsoft/healthvault-ios-sdk/releases/tag/3.0-preview.4)
+- [remove usage of strlen() (#143)](http://github.com/Microsoft/healthvault-ios-sdk/commit/09ea238453ed5c540f91c2c8d32f636264ab9851) 
+- [Remove DynamicEnums (#142)](http://github.com/Microsoft/healthvault-ios-sdk/commit/b6da781deb028576d6914045b3d1508348e2ac89) 
+
+## [3.0-preview.3](https://github.com/Microsoft/healthvault-ios-sdk/releases/tag/3.0-preview.3)
+- [Personal/namalu/add is authenticated property (#141)](http://github.com/Microsoft/healthvault-ios-sdk/commit/d1bbf5d9a4e818fc8b76424f8152b010ef7526cb) 
```

Comment:
Looks auto-generated, so might not be easy to remove the personal/* parts.  You can rename a completed PR, so that might do it.